Documentation/TeleportSystem/Overview.md

```diff
@@ -0,0 +1,88 @@
+# Teleport System
+
+The teleport system is a sub-system of the MRTK that handles teleporting the user when the
+application is using an opaque display. For AR experiences (like HoloLens), the teleportation
+system is not active. For immersive HMD experiences (OpenVR, WMR) the teleport system can
+be enabled.
+
+## Enabling and disabling
+
+The teleport system can be enabled or disabled by toggling the checkbox in its profile:
+
+![Teleport system profile](../../Documentation/Images/TeleportSystem/profile.png)
+
+This can also be done at runtime:
+
+```csharp
+void DisableTeleportSystem()
+{
+    CoreServices.TeleportSystem.Disable();
+}
+
+void EnableTeleportSystem()
+{
+    CoreServices.TeleportSystem.Enable();
+}
+```
+
+## Events
+
+The teleport system exposes events through the [`IMixedRealityTeleportHandler`](xref:Microsoft.MixedReality.Toolkit.Teleport.IMixedRealityTeleportHandler)
```

Comment:
Should we link to teleport pointers in the pointers.md?

Reply:
I think there's some value to this, I haven't quite figured out the content yet, so I'll probably proceed with this change first and we can add the other parts as it comes to me.